lib/filter.js

```diff
@@ -180,6 +181,7 @@ if (Meteor.isClient) {
     return _.map(filterIds, function (filterId) {
       if (_.isUndefined(reactiveTableFilters[filterId])) {
         reactiveTableFilters[filterId] = new ReactiveTable.Filter(filterId);
+        return "";
```

Comment:
Does it still work without this line? This will reintroduce the bug that 0.8.29 was fixing.